src/plugins/messenger/MessengerPreview/components/MessengerTextWithQuickReplies/MessengerTextWithQuickReplies.tsx

```diff
@@ -102,6 +102,7 @@ export const getMessengerTextWithQuickReplies = ({
                                     const { title, image_url, image_alt_text } = quickReply as IFBMTextQuickReply;
                                     label = title;
                                     if (image_url) image = <QuickReplyImage src={image_url} alt={image_alt_text || ""}/>;
+                                    if (config.settings.useQuickReplyTitleAsPostback) (quickReply as IFBMTextQuickReply).payload = title;
```

Comment:
Instead of modifying the object, please just create a variable that is either set to one or the other value and use it there

Reply:
thx, fixed it